spec/helpers/session_helper_spec.rb

```diff
@@ -40,16 +40,22 @@
   end
 
   describe "#show_callback_date" do
-    it "returns the session callback_date value" do
-      session[:registration] = { "callback_date" => Date.new(2000, 10, 1) }
-      expect(show_callback_date).to eq("01 10 2000")
+    it "returns the session callback_date value as a string" do
+      session[:registration] = {
+        "callback_slot" => ApiClient.get_callback_booking_quotas.first.id,
+      }
+      expect(show_callback_time).to be_instance_of(String)
```

Comment:
We should be able to check this is equal to the value stubbed out in the cassette rather than just a non-empty string

Reply:
Might that not get out of synch with what is returned from the api ?